helm/charts/infra/README.md

```diff
@@ -0,0 +1,70 @@
+# Infra Chart
+
+A Helm chart for Infra, an infrastructure access management tool for Kubernetes.
+
+The default deployment installs `infra-server` and accompanying services including a frontend service and a PostgreSQL database. The full list of configuration values can be found with `helm show values infrahq/infra`.
+
+## Upgrade
+
+The upgrade is a standard Helm upgrade. Run the commands below to update the Helm repository and perform the upgrade. Ensure the correct values are specified to avoid any unnecessary changes to the deployment. As in the previous step, these commands assume the Helm release is called `infra`, update as needed.
+
+```bash
+RELEASE_NAME=infra
+RELEASE_NAMESPACE=$(helm status $RELEASE_NAME -o json | jq -r .namespace)
+HELM_ARGS=.. # additional arguments to helm, e.g. --set or --values
+
+### helm repo update infrahq
+helm -n $RELEASE_NAMESPACE upgrade $RELEASE_NAME infrahq/infra $HELM_ARGS
+```
+
+Once upgraded, verify the deployment with `infra login` or the Infra UI. Once verified, the server persistent volume claim should be disabled, `server.persistence.enabled=false`, to fully remove the SQLite database and old database encryption key.
+
+## Rollback
+
+If the upgrade is unsuccessful, the deployment can be rolled back using Helm. The upgrade can be attempted again or you can contact us through GitHub for additional assistance.
```

Comment:
I would move `The upgrade can be attempted again...` to the end of this section and put something like:

```To perform the rollback, run the following commands:```